packages/api/package.json

```diff
@@ -44,6 +44,7 @@
     "chai": "^4.2.0",
     "chai-http": "^4.3.0",
     "cors": "^2.8.5",
+    "crypto-js": "^4.0.0",
```

Comment:
Did we end up using `crypto-js`? I imagine we should clear this out if we did not